test/test-helpers/fake-data.ts

```diff
@@ -400,13 +400,13 @@ export const fakeUploadedFile = async (fileData: Partial<InferCreationAttributes
   const fileName = `${randStr()}${extension}`;
   const kind = sample(SUPPORTED_FILE_KINDS);
   return models.UploadedFile.create({
-    url: fakeUploadedFileURL(kind, fileName),
+    url: fakeUploadedFileURL(fileData?.kind || kind, fileName),
     kind,
     fileSize: randNumber(100, 100000),
```

Comment:
If we do that, we will end up with different file kinds between the `url` and the `kind` columns. Suggested:

```ts
  const kind = fileData?.kind || sample(SUPPORTED_FILE_KINDS);
  return models.UploadedFile.create({
    url: fakeUploadedFileURL(kind, fileName),
    kind,
```